enterprise-suite/templates/alertmanager-deployment.yaml

```diff
@@ -19,11 +19,18 @@ spec:
     matchLabels:
       app: prometheus
       component: alertmanager
+
   template:
     metadata:
       annotations:
         checksum/alertmanager-config: {{ (.Files.Glob "alertmanager/*").AsConfig | sha256sum }}
       labels:
+        app.kubernetes.io/name: {{ template "name" . }}
+        helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```

Comment:
Just curious why the need for the `replace` bit?

Reply:
This is copied right from the helm docs on what to use. But it's because you can't use `+` in label values I believe, and `+` is a valid semver character.